### ISSUE TITLE
Documentation not up-to-date

### ISSUE THREAD

**Issue by Blankarsch** [firebase/FirebaseUI-iOS#45]
Overall great job so far. But please revise your documentation. For example the following code sniped with FirebaseTableViewDataSource in swift:

```
self.dataSource = FirebaseTableViewDataSource(ref: firebaseRef cellClass: YourCustomClass.self cellReuseIdentifier: @"<YOUR-REUSE-IDENTIFIER>" view: self.tableView)

self.dataSource.populateCellWithBlock { (cell: YourCustomClass, obj: NSObject) -> Void in
  // Populate cell as you see fit, like as below
  cell.yourCustomLabel.text = snap.key;
}

self.tableView.dataSource = self.dataSource;

```
There are some mistakes in this code...
- Parameters are separated with commas in swift ( FirebaseTableViewDataSource(...) )
- Swift not using @ for strings (Objective C stuff...)
- Initialization of the variable "snap" is missing
- Semicolons not necessary in swift
- ...

Valid swift code could look like this:

```
self.dataSource = FirebaseTableViewDataSource(ref: firebaseRef, cellClass: YourCustomClass.self, cellReuseIdentifier: "<YOUR-REUSE-IDENTIFIER>", view: self.tableView)

self.dataSource.populateCellWithBlock { (cell: YourCustomClass, obj: NSObject) -> Void in
  let snap = obj as! FDataSnapshot
  // Populate cell as you see fit, like as below
  cell.yourCustomLabel.text = snap.key
}

self.tableView.dataSource = self.dataSource
```

...But nevertheless, this code doesn't work

**Comment by bocodigital** [comment]
Looks like all the swift documentation is incorrect.

**Comment by davideast** [comment]
Yes it's all out of which :cry: . I'll be working on this soon, so sit tight.

**Comment by Blankarsch** [comment]
Cool, thanks for your reply. Looking forward for your alterations!

**Comment by dkbuckley** [comment]
Hi @davideast do you have a timeline for these updates? The above code simply doesn't work anymore and it would be great to actually use this.

**Comment by vkhattar** [comment]
I am currently new to iOS and have been learning swift only. I am unable to find any documentation on how to use the FirebaseUI. I have managed to install it but completely stuck on how to even use this. The manual method of calling methods on the firebase website seems to be more properly documented and user friendly for a swift dev. Can you either provide a sample/steps if possible on how the firebaseUI should help me ! Thanks

**Comment by vinamelody** [comment]
Hi .. did anyone manage to get this snippet work? Xcode is giving error on this line. I can't reference it to my custom class
`self.dataSource.populateCellWithBlock { (cell: YourCustomClass, obj: NSObject) -> Void in`

Thanks!

**Comment by morganchen12** [comment]
@Blankarsch I've just merged a PR updating all these readme docs (#98); if you still have trouble with this feel free to reopen this.
